src/js/base/api.js

```diff
@@ -651,6 +651,190 @@ ripe.Ripe.prototype._getImageOptions = function(options = {}) {
         params.initials = initials;
     }
 
+    if (options.profile !== undefined && options.profile !== null) {

```

Comment:
we already have this one... what's the catch ?

Reply:
There's none, I've missed and I think @NFSS10 didn't see the `params.initials_profile = options.profile.join(",");` above. Right?

Reply:
Yes. it was that

Reply:
Yup, I think that was what happened

Reply:
I guess you can delete `options.initialsProfile` and `options.initialsProfiles` as well, given that `options.profile` is already a list that covers every case.